src/webdriver_utils.py

```
@@ -1,212 +1,171 @@
+import pandas as pd
+from bs4 import BeautifulSoup
+from selenium.common.exceptions import WebDriverException
+from tqdm import tqdm
+from sqlalchemy.exc import IntegrityError
+
+# --- 修正点 1: 必要なモジュールをインポート ---
+from src.db_utils import DatabaseManager
+from src.models import Race, Result
+from src.webdriver_utils import get_webdriver_manager, driver_scope
+
+class DataScraper:
```

Pylint (3.3.7)

[refactor] Too few public methods (1/2)

(R0903)